charts/sonarqube/README.md

```diff
@@ -4,13 +4,13 @@ Code better in up to 27 languages. Improve Code Quality and Code Security throug
 
 ## Introduction
 
-This chart bootstraps an instance of the latest SonarQube version with a PostgreSQL database. A helm chart is also available for the [LTS version](../sonarqube-lts).
+This chart bootstraps an instance of the latest SonarQube version with a PostgreSQL database. A helm chart containing the SonarQube LTS version can be installed following the steps outlined [here](#installing-the-lts-chart).
```

Comment:
Similar suggestion as for sonarqube-dce:
```suggestion
This chart bootstraps an instance of the latest SonarQube version with a PostgreSQL database.

The latest version of the chart installs the latest SonarQube version. To install the version of the chart for SonarQube 9.9 LTS, please read the section [below](#installing-the-lts-chart). Deciding between LTS and Latest? [This may help](https://www.sonarsource.com/products/sonarqube/downloads/lts/)
```